repo/create-repo.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+log() {
+    echo "$@" >&2
+}
+
+SCRIPT_DIR="$(dirname "$0")"
+source "${SCRIPT_DIR}/vars.sh"
+
+REPO_CONFIG="${SCRIPT_DIR}/repo-config.ttl"
+REPO_URL="${GRAPHDB_URL}/rest/repositories/${REPO_ID}"
```

Comment:
If the repo name is in the config ttl file the  rest is redundant and conflicts if the name from the scrit conflicts with the one in the ttl.

Reply:
agree, I made the repo-config.ttl file to be a template. The script now uses `sed` to fill in the REPO_ID in the template.